test/puppetlabs/comidi_test.clj

```diff
@@ -271,14 +326,26 @@
 
     (handler {:uri "/bar/something" :request-method :get})
     (is (= (-> test-atom deref :route-info :route-id) "bar-:bar"))
+    (is (= (-> test-atom deref :route-metadata) route-meta))
+
+    (handler {:uri "/false" :request-method :get})
+    (is (= (-> test-atom deref :route-info :route-id) "false"))
+    (is (= (-> test-atom deref :route-metadata) route-meta))
+
+    (handler {:uri "/true" :request-method :get})
+    (is (= (-> test-atom deref :route-info :route-id) "true"))
+    (is (= (-> test-atom deref :route-metadata) route-meta))
+
+    (handler {:uri "/wat" :request-method :get})
+    (is (= (-> test-atom deref :route-info :route-id) "*"))
```

Comment:
Yeah, getting the schemas right is definitely part of it. I can see use-cases for using "true" as a catch-all and if you're approaching comidi with knowledge of bidi I think you might expect to be allowed to use booleans. 

The main motivator for me is that it helps partial migration of a compojure app to comidi (something like the below) and allows comidi to support hand-crafting the bidi routes:

```clojure
(comidi/context "/api"
  (comidi/ANY "/app1" request ...)
  (comidi/ANY true request ((compojure/context "/legacy-app" [] ...) request)))
```